.eslintrc.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  'env': {
```

Comment:
Si está como JS vamos derecho con algo como:
```suggestion
  env: {
```
En todos los keys.